### PR TITLE
add ci job  for commatrix e2e-tests

### PR DESCRIPTION
**Pull request by aabughosh**

### CODE REVIEW

**Comment by aabughosh** [comment]
/pj-rehearse pull-ci-openshift-kni-commatrix-main-e2e-aws-commatrix-run

**Comment by aabughosh** [comment]
/pj-rehearse pull-ci-openshift-kni-commatrix-main-e2e-aws-commatrix-run

**Comment by aabughosh** [comment]
/pj-rehearse pull-ci-openshift-kni-commatrix-main-e2e-aws-commatrix-run

**Comment by aabughosh** [comment]
/pj-rehearse pull-ci-openshift-kni-commatrix-main-e2e-aws-commatrix-run

**Comment by aabughosh** [comment]
/pj-rehearse pull-ci-openshift-kni-commatrix-main-e2e-aws-commatrix-run

**Comment by aabughosh** [comment]
/pj-rehearse pull-ci-openshift-kni-commatrix-main-e2e-aws-commatrix-run

**Comment by aabughosh** [comment]
/pj-rehearse pull-ci-openshift-kni-commatrix-main-e2e-aws-commatrix-run

**Comment by aabughosh** [comment]
/pj-rehearse pull-ci-openshift-kni-commatrix-main-e2e-aws-commatrix-run

**Comment by aabughosh** [comment]
/pj-rehearse pull-ci-openshift-kni-commatrix-main-e2e-aws-commatrix-run

**Comment by aabughosh** [comment]
/pj-rehearse pull-ci-openshift-kni-commatrix-main-e2e-aws-commatrix-run

**Comment by aabughosh** [comment]
/pj-rehearse pull-ci-openshift-kni-commatrix-main-e2e-aws-commatrix-run

**Comment by aabughosh** [comment]
@droslean

**Comment by aabughosh** [comment]
/pj-rehearse pull-ci-openshift-kni-commatrix-main-e2e-aws-commatrix-run

**Comment by aabughosh** [comment]
/pj-rehearse pull-ci-openshift-kni-commatrix-main-e2e-aws-commatrix-run

**Comment by aabughosh** [comment]
/pj-rehearse pull-ci-openshift-kni-commatrix-main-e2e-aws-commatrix-run

**Comment by aabughosh** [comment]
/pj-rehearse pull-ci-openshift-kni-commatrix-main-e2e-aws-commatrix-run

**Comment by aabughosh** [comment]
/pj-rehearse pull-ci-openshift-kni-commatrix-main-e2e-aws-commatrix-run

**Comment by aabughosh** [comment]
/pj-rehearse pull-ci-openshift-kni-commatrix-main-e2e-aws-commatrix-run

**Comment by aabughosh** [comment]
/pj-rehearse pull-ci-openshift-kni-commatrix-main-e2e-aws-commatrix-run

**Comment by aabughosh** [comment]
/pj-rehearse pull-ci-openshift-kni-commatrix-main-e2e-aws-commatrix-run

**Comment by aabughosh** [comment]
/pj-rehearse pull-ci-openshift-kni-commatrix-main-e2e-aws-commatrix-run

**Comment by aabughosh** [comment]
/pj-rehearse pull-ci-openshift-kni-commatrix-main-e2e-aws-commatrix-run

File: ci-operator/config/openshift-kni/commatrix/openshift-kni-commatrix-main.yaml [hunk 1]
```diff
@@ -14,6 +24,30 @@ tests:
   commands: make lint
   container:
     from: src
+- as: e2e-aws-commatrix-run
```
Comment (sabinaaledort) [inline ci-operator/config/openshift-kni/commatrix/openshift-kni-commatrix-main.yaml:27]: commatrix-run -> commatrix-e2e-tests

**Comment by aabughosh** [comment]
/pj-rehearse pull-ci-openshift-kni-commatrix-main-e2e-aws-commatrix-run

**Comment by aabughosh** [comment]
/pj-rehearse pull-ci-openshift-kni-commatrix-main-e2e-aws-commatrix-run-tests

**Comment by aabughosh** [comment]
/pj-rehearse pull-ci-openshift-kni-commatrix-main-commatrix-e2e-tests

**Pull request merged**